mitosheet/mitosheet/code_chunks/export_to_file_code_chunk.py

```diff
@@ -18,16 +18,26 @@ def get_format_code(state: State) -> list:
     formats = state.df_formats
     for sheet_name, format in zip(state.df_names, formats):
         # If there is no formatting, we skip trying to access the colors
-        if format.get('headers') is None:
-            continue
-        
-        # Prepares the header font color and background color to be used in the API call
-        # So either they should be surrounded by quotes, or they should be None
-        params = {}
-        if (format['headers'].get('backgroundColor') is not None):
-            params['header_background_color'] = format["headers"]["backgroundColor"]
-        if (format['headers'].get('color') is not None):
-            params['header_font_color'] = format["headers"]["color"]
+        if format.get('headers') is not None:
+            # Prepares the header font color and background color to be used in the API call
+            # So either they should be surrounded by quotes, or they should be None
+            params = {}
```

Comment:
This probably all could be rewritten as the following:
```
params = {
     'header_background_color': format.get('headers', {}).get('backgroundColor')
     'header_font_color': format.get('headers', {}).get('color'),
     ....

}
```

The two tricks here:
1. Creating the param in one place, which is clean. 
2. Using default arguments to .get to define a new dict, so you can then continue and look up on it if it doesn't exist. 

I think this could cut like 10 or so lines out of this function.

Reply:
Ahh that is the syntax I was looking for! That's awesome I'll definitely add it